### PR TITLE
feat: intergration of Rabby wallet

### PR DESCRIPTION
**Pull request by vvvvvv1vvvvvv**
### Description
This PR add intergration of Rabby wallet, Rabby will add into the list if user installed Rabby

image

More about Rabby wallet: https://rabby.io/

### Checklist
- [x] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [x] The box that allows repo maintainers to update this PR is checked
- [x] I tested locally to make sure this feature/fix works
- [x] This PR passes the Circle CI checks

### CODE REVIEW

File: packages/injected/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/injected-wallets",
-  "version": "2.0.9",
+  "version": "2.0.10",
```
Comment (taylorjdawson) [inline packages/injected/package.json:3]:
```suggestion
  "version": "2.0.10-alpha.1",
```

File: packages/injected/src/icons/rabby.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+export default `<svg width="320" height="320" viewBox="0 0 320 320" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (taylorjdawson) [inline packages/injected/src/icons/rabby.ts:1]: @vvvvvv1vvvvvv We like to have the icons with 100% width and height, thanks!
```suggestion
export default `<svg width="100%" height="100%" viewBox="0 0 320 320" fill="none" xmlns="http://www.w3.org/2000/svg">
```

File: packages/injected/src/types.ts [hunk 3]
```diff
@@ -40,7 +40,8 @@ export enum ProviderIdentityFlag {
   OneInch = 'isOneInchIOSWallet',
   Tokenary = 'isTokenary',
   Tally = 'isTally',
-  BraveWallet = 'isBraveWallet'
+  BraveWallet = 'isBraveWallet',
+  Rabby = 'isRabby',
```
Comment (taylorjdawson) [inline packages/injected/src/types.ts:44]: We have `trailingComma` disabled. I would run `yarn format` just to make sure your files are correctly formatted
```suggestion
  Rabby = 'isRabby'
```

**Review by taylorjdawson — CHANGES_REQUESTED** [review]
@vvvvvv1vvvvvv  A few comments but looks good, thanks for the contribution!

**Comment by vvvvvv1vvvvvv** [comment]
@taylorjdawson all suggestions fixed

**Comment by aaronbarnardsound** [comment]
Thanks @vvvvvv1vvvvvv this looks good. We'll get this merged and released later this week.

File: packages/injected/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/injected-wallets",
-  "version": "2.0.11-alpha.1",
+  "version": "2.0.11-alpha.2",
```
Comment (Adamj1232) [inline packages/injected/package.json:3]:
```suggestion
  "version": "2.0.12-alpha.1",
```

**Comment by Adamj1232** [comment]
@vvvvvv1vvvvvv we had a recent release so there is a conflict and a note to update the version otherwise I can get this merged as soon as this is completed

**Review by Adamj1232 — APPROVED** [review]
Nice, thank you for the PR!

**Pull request merged**